src/SelectableBox/SelectableBoxSet.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Form from '../Form';
+
+const INPUT_TYPES = [
+  'radio',
+  'checkbox',
+];
+
+const SelectableBoxSet = ({
```

Comment:
Should this be wrapped with `React.forwardRef` so consumers can pass a ref if needed?

Reply:
Right, component is wrapped